packages/components/src/components/Alerts/collapse.tsx

```diff
@@ -20,6 +20,7 @@ import type { AlertProps } from '../Alert/types';
 import styles from './collapse.module.scss';
 
 const { Paragraph, Text } = Typography;
+const InnerWidth = window.innerWidth;
```

Comment:
Use camelCase for variables.

```suggestion
const innerWidth = window.innerWidth;
```